apps/console/src/features/users/components/user-roles-list.scss

```diff
@@ -0,0 +1,5 @@
+.list-user-roles-section {
```

Comment:
shall we add a header for this file

Reply:
Fixed with 5f3e39c54d9a02658bf4c6d7abd3f538899828ea